### DISCUSSION TITLE
Giro stabilized motors

### DISCUSSION THREAD

**Discussion by Taboaco** [iNavFlight/inav#7704] [General]
Hello guys, one last try here, I need to set a wing with differential thrust on inav, so far so good, but I really need the motors to be giro or accelerometer stabilized to avoid “ Dutch roll” kind of turbulence, just lock my yaw like a quad does, I believe it will benefit everyone that flies those kind of FPV tween motor wings and will make the FPV experience way more enjoyable and less shaky, I really need this due a project I have in a not sweep wing with no vertical stabilizer fins, I don’t know it I mix an yaw or gimbal with motors,… but I was unable to do it so!
Please help!
Thanks

**Comment by MrD-RC** [comment]
This should gave been an issue, so I have moved it over.
